app/src/main/java/com/internship/move/LoginFragment.kt

```diff
@@ -0,0 +1,28 @@
+package com.internship.move
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.internship.move.databinding.FragmentLoginBinding
+import com.zhuinden.fragmentviewbindingdelegatekt.viewBinding
+
+class LoginFragment : Fragment(R.layout.fragment_login) {
+
+    private val binding by viewBinding(FragmentLoginBinding::bind)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initButtons()
+    }
+
+    private fun initButtons() {
```

Comment:
Personally I see `initButtons` as being a bit too specific. Usuallty we go with something like `initListeners`. This is beacause many types of views can have listeners and if a particular view has a more complex init setup, we create a method for that view